readme.txt

```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
+Easy Digital Downloads has been translated into more than two dozen languages and new translation are being added all the time, letting store owners have a natural, native experience, regardless of where you live. If your language is not available, we welcome translation contributions! See our [FAQ](http://docs.easydigitaldownloads.com/article/1082-what-spoken-languages-does-easy-digital-downloads-support?utm_source=docs&utm_medium=description_tab&utm_content=languages&utm_campaign=readme).
 
-More information at [Easy Digital Downloads.com](https://easydigitaldownloads.com/?utm_source=home&utm_medium=description_tab&utm_content=home&utm_campaign=readme).
+= Payment gateways for everyone =
 
-**Sell and Track Software License Keys**
+The internet has made it possible for anyone to sell their products to a world-wide audience. No matter where you live, Easy Digital Downloads can work for you. We offer integrations for the most common merchant processors and, through 3rd party extensions, support for many, many more as well.
 
-[Software Licensing](https://easydigitaldownloads.com/downloads/software-licensing/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) for Easy Digital Downloads provides a complete license key generation, activation, and checking system. It allows you to provide license keys along with your digital products that can then be used to properly license and activate the buyer's copy of the software.
+Payment gateways supported in the core, free plugin:
 
-Includes a complete integration pack for delivering automatic upgrades for premium WordPress plugins and themes.
+* PayPal Standard
+* Amazon Payments
 
-**Sell Recurring Subscriptions**
+Payment gateways supported through free or premium extension:
 
-Using our [Recurring Payments](https://easydigitaldownloads.com/downloads/recurring-payments/?utm_source=extension&utm_medium=description_tab&utm_content=recurring-payments&utm_campaign=readme) add-on, you can easily sell recurring subscriptions for your products and/or services.
+* Stripe
+* PayPal Pro
+* PayPal Express
+* PayPal Payments Advanced
+* PayPal Adaptive Payments
+* Braintree
+* Authorize.net
+* 2Checkout
+* Recurly
+* SOFORT Banking
+* NETbilling
+* BitPay
+* Coinbase
+* Moneybookers / Skrill
+* Payza
+* MercadoPago
+* PagSeguro
 
-**Build Up Your Email Subscribers**
+See our [gateways category](https://easydigitaldownloads.com/downloads/category/gateways/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) in the extensions catalogue for a complete list of supported gateways. There are also a large number of additional gateways developed and supported by [3rd party developers](https://easydigitaldownloads.com/3rd-party-extensions/#gateways).
```

Comment:
the `utm_content` here should be `gateways`, so we can focus on traffic directly going to gateways.